libpkg/scripts.c

```diff
@@ -198,6 +198,9 @@ pkg_script_run(struct pkg * const pkg, pkg_script type)
 			}
 
 			if (WEXITSTATUS(pstat) != 0) {
+				if (WEXITSTATUS(pstat) == 3)
```

Comment:
I think this should be:
`if (WEXITSTATUS(pstat) != 0 || WEXITSTATUS(pstat) == 3)`

But, as I think more about this, I'm not sure this is sufficient, as the 'script failed' will still be printed (although it did not fail).